internal/collector/collector_test.go

```diff
@@ -355,7 +355,7 @@ var _ = Describe("Factory#NewPodSpec Add Cloudwatch STS Resources", func() {
 				podSpec := *factory.NewPodSpec(nil, logging.ClusterLogForwarderSpec{
 					Outputs:   outputs,
 					Pipelines: pipelines,
-				}, "1234", "", tls.GetClusterTLSProfileSpec(nil))
+				}, "1234", "", tls.GetClusterTLSProfileSpec(nil), []string{})
```

Comment:
Aside: it's safe to pass 'nil' for an empty slice, all the slice operators  (append, len  etc.) treat nil and [] as the same.
(note this does not apply to empty maps...)